docs/data-sources/security_monitoring_rules.md

```diff
@@ -1,49 +1,75 @@
-    -   `name`: The name of the rule.
-    -   `options`: Options on the rule
-        -   `evaluation_window`: A time window is specified to match when at least one of the cases matches true. This is a sliding window and evaluates in real time. Allowed values: `0,60,300,600,900,1800,3600,7200`.
-        -   `keep_alive`: Once a signal is generated, the signal will remain “open” if a case is matched at least once within this keep alive window. Allowed values: `0,60,300,600,900,1800,3600,7200,10800,21600`.
-        -   `max_signal_duration`: A signal will "close" regardless of the query being matched once the time exceeds the maximum duration. This time is calculated from the first seen timestamp. Allowed values: `0,60,300,600,900,1800,3600,7200,10800,21600,43200,86400`.
-    -   `query`: Queries for selecting logs which are part of the rule.
-        -   `aggregation`: The aggregation type. Allowed values: `count`, `cardinality`, `sum`, `max`
-        -   `distinctFields`: Fields for which the cardinality is measured.
-        -   `groupByFields`: Fields to group by.
-        -   `metric`: The target field to aggregate over when using the sum or max aggregations.
-        -   `name`: Name of the query.
-        -   `query`: Query to run on logs.
-    -   `tags`: Tags for generated signals.
+## Schema
+
+### Optional
+
+- **default_only_filter** (Boolean, Optional) Limit the search to default rules
+- **id** (String, Optional) The ID of this resource.
+- **name_filter** (String, Optional) A rule name to limit the search
+- **tags_filter** (List of String, Optional) A list of tags to limit the search
+- **user_only_filter** (Boolean, Optional) Limit the search to user rules
+
+### Read-only
+
+- **rule_ids** (List of String, Read-only) List of IDs of the matched rules.
+- **rules** (List of Object, Read-only) List of rules. (see [below for nested schema](#nestedatt--rules))
+
+<a id="nestedatt--rules"></a>
+### Nested Schema for `rules`
+
+- **case** (List of Object) (see [below for nested schema](#nestedobjatt--rules--case))
+- **enabled** (Boolean)
+- **message** (String)
+- **name** (String)
+- **options** (List of Object) (see [below for nested schema](#nestedobjatt--rules--options))
+- **query** (List of Object) (see [below for nested schema](#nestedobjatt--rules--query))
+- **tags** (List of String)
+
+<a id="nestedobjatt--rules--case"></a>
+### Nested Schema for `rules.case`
+
+- **condition** (String)
+- **name** (String)
+- **notifications** (List of String)
+- **status** (String)
+
+
+<a id="nestedobjatt--rules--options"></a>
+### Nested Schema for `rules.options`
+
+- **evaluation_window** (Number)
+- **keep_alive** (Number)
+- **max_signal_duration** (Number)
+
+
+<a id="nestedobjatt--rules--query"></a>
+### Nested Schema for `rules.query`
+
+- **aggregation** (String)
+- **distinct_fields** (List of String)
+- **group_by_fields** (List of String)
+- **metric** (String)
+- **name** (String)
+- **query** (String)
```

Comment:
We're missing a lot of descriptions here, let's fix that

Reply:
I was looking into this one. The schema for this is shared and referenced both in the data source and the resource. The resource seems to show up as expected. 

I think we're possibly hitting a version of this issue - https://github.com/hashicorp/terraform-plugin-docs/issues/ 28, but I have to investigate a bit more.

Reply:
I see. IIUC, it's all because of the `computed: True` that changes how it's represented internally in terraform right ?

Reply:
Hmm yeah. I just removed `Computed` locally and regenerated and all the descriptions are there